pkg/webhook/merge_on_review_approval_and_status_checks.go

```diff
@@ -40,27 +40,41 @@ func (h *autoMerger) HandleEvent(w http.ResponseWriter, payload interface{}, ghC
 		return h.handlePullRequestEvent(w, event, ghClientFunc)
 	case *github.StatusEvent:
 		return h.handleStatusEvent(w, event, ghClientFunc)
+	case *github.PullRequestReviewEvent:
+		return h.handlePullRequestReviewEvent(w, event, ghClientFunc)
 	default:
 		return nil
 	}
 }
 
+func (h *autoMerger) handlePullRequestReviewEvent(w http.ResponseWriter, event *github.PullRequestReviewEvent, ghClientFunc GitHubIntegrationsClientFunc) error {
```

Comment:
If you handle the approval here directly and somewhere else to add the label which then in turn triggers the other handler watching on new labe;s isn't it then merged twice ? guess not, but I dont understand it really probably ;-)